### ISSUE TITLE
Improve Kappa value for MLP

### ISSUE THREAD

**Issue by bmmidei** [ac4391/EssayGrader#8]
Consider using batch norm, dropout to improve accuracy. Other options include: ensuring an even distribution of scores in the training data, tweaking number of PCA components, tweaking model hyperparams.

**Comment by bmmidei** [comment]
Appears to improve with larger MLP network. Approx [1000, 300] hidden layer sizes. Batch size 16
Essay sets still need to be trained separately.
